fmn/rules/utils.py

```diff
@@ -67,6 +72,32 @@ def get_fas(config):
     return _FAS
 
 
+def _paginate_pagure_data(url, params, key):
```

Comment:
@pypingou, do you know what @jeremycline is talking about here?

> the staging Pagure instance seems to have issues with packages with groups as committers or something

Jeremy, do you have more details about what's failing here?